scripts/unixbench/virtual_application.txt

```diff
@@ -0,0 +1,29 @@
+# Parameters for this Virtual Application (Application Instance - AI) type should
+# be set on YOUR private configuration configuration file, including the ones 
+# commented.
+
+[AI_TEMPLATES : UNIXBENCH] 
```

Comment:
Please also add a section to `configs/templates/PUBLIC_application_submitters.txt`. I know we don't have the full set of applications in there, but let's make it a habit anyway.

Example PR also coming for that.